docker/rootless/etc/templates/app.ini

```diff
@@ -0,0 +1,57 @@
+APP_NAME = $APP_NAME
+RUN_USER = $RUN_USER
+RUN_MODE = $RUN_MODE
+
+[repository]
+ROOT = $GITEA_WORK_DIR/git/repositories
+
+[repository.local]
+LOCAL_COPY_PATH = $GITEA_WORK_DIR/data/tmp/local-repo
```

Comment:
IIRC the files in here do not need to persist. Let's use a directory in `/tmp` instead?

Reply:
I added the variable GITEA_TEMP from where all temporary folder should be relative.